google/resource_cloud_build_trigger.go

```diff
@@ -291,7 +291,7 @@ Default time is ten minutes (600s).`,
 			"ignored_files": {
 				Type:     schema.TypeList,
 				Optional: true,
-				Description: `ignoredFiles and includedFiles are file glob matches using http://godoc/pkg/path/filepath#Match
+				Description: `ignoredFiles and includedFiles are file glob matches using https://golang.org/pkg/path/filepath/#Match
```

Comment:
Unfortunately, this is one of our autogenerated resources, so this changes need to be done in Magic Modules. I created a PR [here](https://github.com/GoogleCloudPlatform/magic-modules/pull/2963), but if you want contributor credit, please let me know and I can merge in your PR first so it'll show up in TPG as your change. You'd just have to remove the vendor files from the PR. Otherwise, if you don't care, just close this PR out and I'll take it as indication to submit my PR.